### PR TITLE
Powerlinks

### PR DESCRIPTION
**Pull request by markbrocato**

### CODE REVIEW

File: packages/react-storefront/service-worker/bootstrap.js [hunk 1]
```diff
@@ -354,14 +358,23 @@ workbox.routing.registerRoute(matchRuntimePath, async context => {
     const cacheName = getAPICacheName(apiVersion, url.pathname)
     const cacheOptions = { ...runtimeCacheOptions, cacheName }
 
+    console.log('cacheOptions.cacheName', cacheOptions.cacheName)
```
Comment (dijs) [inline packages/react-storefront/service-worker/bootstrap.js]: Were you planning on leaving these comments?

File: packages/react-storefront/src/registerServiceWorker.js [hunk 2]
```diff
@@ -48,9 +56,7 @@ export default function register() {
     return
   }
 
-  const installSW = process.env.NODE_ENV === 'production' || process.env.MOOV_SW === 'true'
```
Comment (dijs) [inline packages/react-storefront/src/registerServiceWorker.js]: So we won't need to use `yarn start:sw` anymore?

File: packages/react-storefront/service-worker/bootstrap.js (hunk 1 above)
Comment (markbrocato) [inline packages/react-storefront/service-worker/bootstrap.js]:
```suggestion
```

Comment (markbrocato) [reply packages/react-storefront/service-worker/bootstrap.js]: Removed

**Pull request merged**